src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.jsx

```diff
@@ -1,7 +1,122 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 
-const ClaimsAndAppeals = () => {
-  return <h2>Claims & appeals</h2>;
+import backendServices from '~/platform/user/profile/constants/backendServices';
+import {
+  createIsServiceAvailableSelector,
+  selectProfile,
+} from '~/platform/user/selectors';
+
+import {
+  getAppealsV2 as getAppealsAction,
+  getClaimsV2 as getClaimsAction,
+} from '~/applications/claims-status/actions';
+
+import ClaimsAndAppealsCTA from './ClaimsAndAppealsCTA';
+import HighlightedClaimAppeal from './HighlightedClaimAppeal';
+import useOpenClaimsAppealsCount from './hooks/useOpenClaimOrAppealCount';
+import useHighlightedClaimOrAppeal from './hooks/useHighlightedClaimOrAppeal';
+
+const ClaimsAndAppeals = ({
+  appealsData,
+  claimsData,
+  // for some unit testing purposes, we want to prevent this component from
+  // making API calls which kicks off a chain of events that results in the
+  // component always showing a loading spinner. I do not like this approach.
+  dataLoadingDisabled = false,
+  loadAppeals,
+  loadClaims,
+  shouldLoadAppeals,
+  shouldLoadClaims,
+  shouldShowLoadingIndicator,
+  userFullName,
+}) => {
+  React.useEffect(
+    () => {
+      if (!dataLoadingDisabled && shouldLoadAppeals) {
+        loadAppeals();
+      }
+    },
+    [dataLoadingDisabled, loadAppeals, shouldLoadAppeals],
+  );
+
+  React.useEffect(
+    () => {
+      if (!dataLoadingDisabled && shouldLoadClaims) {
+        loadClaims();
+      }
+    },
+    [dataLoadingDisabled, loadClaims, shouldLoadClaims],
+  );
+
+  // the most recently updated claim or appeal that has been updated in the past
+  // 30 days
+  const highlightedClaimOrAppeal = useHighlightedClaimOrAppeal(
+    appealsData,
+    claimsData,
+  );
+
+  // the total number of open claims and appeals, no matter when they were last
+  // updated
+  const openClaimsOrAppealsCount = useOpenClaimsAppealsCount(
+    appealsData,
+    claimsData,
+  );
+
+  if (!shouldLoadAppeals && !shouldLoadClaims) {
+    return null;
+  }
+
+  if (shouldShowLoadingIndicator) {
+    return <LoadingIndicator />;
+  }
+
+  if (highlightedClaimOrAppeal || openClaimsOrAppealsCount > 0) {
+    return (
+      <div>
+        <h2>Claims & appeals</h2>
```

Comment:
Should we be using `&amp;` here instead of `&`?

Reply:
hmm... this renders fine. I think because the doctype is html5 we can use it without being encoded https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp